### PR TITLE
Fix style to main

### PR DESCRIPTION
**Pull request by aronnoordam**
#252

### CODE REVIEW

File: tests/test_model.py [hunk 1]
```diff
@@ -3462,3 +3470,193 @@ def test_split_model_part_expected_raise(self, create_default_2d_soil_material:
...
+        msg = "No name is specified for the json file."
+        with pytest.raises(ValueError, match=msg):
+            model_copy.finalise(input_folder="input_files")
+
+        # set json filename to a wrong one
+        model_copy = deepcopy(model)
+        model_copy.output_settings[-1].output_name = "json_nodal_displacements"
+
+        expected_path = Path(
+            'input_dir') / model_copy.output_settings[-1].output_dir / model_copy.output_settings[-1].output_name
```
Comment (brunozc) [inline tests/test_model.py:3526]:
```suggestion
        expected_path = Path('input_dir', model_copy.output_settings[-1].output_dir, model_copy.output_settings[-1].output_name)
```

File: tests/test_stem.py [hunk 2]
```diff
@@ -585,3 +618,110 @@ def test_set_output_name_new_stage_with_json_output(self, create_default_model:
 
         # check if the output name is set correctly
         assert new_stage.output_settings[0].output_name == "json_output_stage_3"
+
+    def test_finalise_stage_json_output(self):
+        """
+        Test the finalise method of the Stem class with JsonOutputParameters and checks the order of the keys in the dictioary.
```
Comment (brunozc) [inline tests/test_stem.py:624]: ``
```suggestion
        Test the finalise method of the Stem class with JsonOutputParameters and checks the order of the keys in the dictionary.
```

**Review by brunozc — CHANGES_REQUESTED** [review]

File: benchmark_tests/test_multi_stage_umat/test_multi_stage_umat.py [hunk 3]
```diff
@@ -0,0 +1,279 @@
...
+SHOW_RESULTS = False
+
+
+@pytest.mark.skipif(sys.platform == "linux", reason="linear elastic umat is currently not available for linux")
+def test_stem():
+    """
+    Test STEM: 2D block with distributed loading with multistage for the umat using umat and changing the stiffness
+    of the material in the second stage (halved).
+
+    Note that currently, 2D elements are written with the incremental formulation, and 3D with the full formulation. Different results in multistage analyses are expected
```
[the remark this answers is not in the document]
Comment (aronnoordam) [reply benchmark_tests/test_multi_stage_umat/test_multi_stage_umat.py:44]: it refers to the 2D plane strain linear elastic constitutive law, 

the stressvector in this law is calculated as follows:

`    mDeltaStrainVector = rValues.GetStrainVector() - mStrainVectorFinalized;

    Matrix C;
    this->CalculateElasticMatrix(C, rValues);

    // Incremental formulation
    noalias(mStressVector) = mStressVectorFinalized + prod(C, mDeltaStrainVector);

    rStressVector = mStressVector;`

[the remark this answers is not in the document]
Comment (aronnoordam) [reply benchmark_tests/test_multi_stage_umat/test_multi_stage_umat.py:44]: i will change the comment

File: stem/model.py [hunk 4]
```diff
@@ -1549,3 +1618,85 @@ def split_model_part(self, from_model_part_name: str, to_model_part_name: str, g
...
+                # adjust the nodal outputs in the right order
+                for node_id in output_model_part.mesh.nodes.keys():
+                    node_key = f"NODE_{node_id}"
+                    new_json[node_key] = json_data_tmp[node_key]
+
+                # write back the json file
+                with open(json_file_path, "w") as outfile:
+                    json.dump(new_json, outfile, indent=2)
+
+    def finalise(self, input_folder: str):
```
[the remark this answers is not in the document]
Comment (aronnoordam) [reply stem/model.py:1691]: maybe in the future more stuff has to be finalised, now the structure is already in place

File: benchmark_tests/test_multi_stage_dynamic_to_static/test_multistage_dynamic_to_static.py [hunk 5]
```diff
@@ -0,0 +1,248 @@
...
+    input_folder = "benchmark_tests/test_multi_stage_dynamic_to_static/inputs_kratos"
+    stem = Stem(model_stage_1, input_folder)
+
+    # create new stage
+    delta_time_stage_2 = 0.01
+    model_stage_2 = stem.create_new_stage(delta_time_stage_2, 0.03)
+
+    # Set up solver settings for the new stage
+    model_stage_2.project_parameters.settings.solution_type = SolutionType.QUASI_STATIC
+    model_stage_2.project_parameters.settings.rayleigh_k = 0.0
```
[the remark this answers is not in the document]
Comment (aronnoordam) [reply benchmark_tests/test_multi_stage_dynamic_to_static/test_multistage_dynamic_to_static.py:172]: kraton will perform a dynamic calculation with only a damping matrix, and now mass. Maybe we should prevent this behaviour to never use this solver?

File: tests/test_model.py (hunk 1 above)
Comment (aronnoordam) [reply tests/test_model.py:3526]: this follows from the linter

File: benchmark_tests/test_multi_stage_dynamic_to_static/test_multistage_dynamic_to_static.py (hunk 5 above)
[the remark this answers is not in the document]
Comment (brunozc) [reply benchmark_tests/test_multi_stage_dynamic_to_static/test_multistage_dynamic_to_static.py:172]: Create an issue to address this

**Review by brunozc — APPROVED** [review]

**Pull request merged**